application/clicommands/ScanCommand.php

```diff
@@ -4,28 +4,36 @@
 
 namespace Icinga\Module\X509\Clicommands;
 
+use DateTime;
 use Icinga\Application\Logger;
 use Icinga\Module\X509\CertificateUtils;
 use Icinga\Module\X509\Command;
 use Icinga\Module\X509\Hook\SniHook;
 use Icinga\Module\X509\Job;
+use InvalidArgumentException;
+use ipl\Sql\Connection;
+use ipl\Sql\Expression;
 
 class ScanCommand extends Command
 {
     /**
      * Scans IP and port ranges to find X.509 certificates.
      *
      * This command starts scanning the IP and port ranges which belong to the job that was specified with the
-     * --job parameter.
+     * --job parameter and optional --since-last-scan and --since-last-seen parameters.
      *
      * USAGE
      *
-     * icingacli x509 scan --job <name>
+     * icingacli x509 scan --job <name> --since-last-scan <duration> --since-last-seen <duration>
```

Comment:
Maybe something like this looks much better:
```
/**
  * Scans IP and port ranges to find X.509 certificates.
  *
  * USAGE
  *
  * icingacli x509 scan [options]
  *
  * OPTIONS:
  *
  *   --job=<name>                  Scans IP and port combinations belonging to the specified job.
  *   --since-last-scan=<datetime>  Scans only the targets whose last scan is older than the specified
  *                                 date/time, which can also be an English textual date time description
  *                                 like "2 days". (Optional)
  *   --since-last-seen=<datetime>  Purges certificates and targets whose last sighting date time is older than the
  *                                 specified date/time, which can also be an English textual date time description
  *                                 like "1 day" or "-3 days". (Optional)
  */
```